internal/git/util_test.go

```diff
@@ -289,9 +289,9 @@ func Test_GetLatestManifests(t *testing.T) {
 				Spec: v1alpha1.GitSyncSpec{
 					GitSource: v1alpha1.GitSource{
 						GitLocation: v1alpha1.GitLocation{
-							RepoUrl:        "https://github.com/numaproj/numaflow.git",
-							TargetRevision: "main",
-							Path:           "config/namespace-install",
+							RepoUrl:        "https://github.com/numaproj-labs/numaplane.git",
+							TargetRevision: "controlledTestsPathEn",
```

Comment:
I guess you'll have to do a follow up commit unfortunately in which you need to change this to "main", huh? (sorry)

Reply:
yes right once the code gets merged to main then i will make a follow up pr .Please dnt delete this branch post merge